src/content/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data.mdx

```diff
@@ -50,244 +50,226 @@ You can use tags in the UI to filter down to relevant data. Here is an example:
 
-Format requirements and limits for tags:
+When you add tags via the UI or API, this occurs at the entity level. This means that, for example, if you have multiple APM agents monitoring a single entity and apply a tag via the UI, that tag is available for all data coming from that entity.
 
-* The UI has limits on the total number of tags it can display per entity:
-  * Limit of 100 for tags reported by default or added via agent/integration configuration.
-  * Limit of 100 for custom tags added via UI or API.
-* The following are maximum character length for tags:
-  * Key: 128 characters
-  * Value: 256 characters
-* Allowed characters:
-  * Characters must be UTF-8.
-  * When [using NerdGraph to add tags](/docs/apis/nerdgraph/examples/nerdgraph-tagging-api-tutorial), a dash (`-`) in a tag key is interpreted as a minus symbol. If your tag key has a dash, use back ticks around it (like `` `key-name` ``).
+### Add tags [#add-tags]
 
-## Best practices and tips [#tag-best-practices]
+1. Go to [one.newrelic.com](https://one.newrelic.com) and select a monitored entity, like a monitored app or host.
+2. Near the entity's name at the top of the page, select the **See metadata and manage tags** menu.
+3. In the **Add a new tag...** field of the pop-up menu, add a tag with this format: `accountId: 123456789`.
+  <img
+  title="Add tags in New Relic"
+  alt="Add tags in New Relic"
+  src={tagManagement}
+  />
 
-Before [adding tags](#tag-sources), we recommend first seeing how our default tags work for you. When you're ready to add your own tags, review the following best practices tips to get the most out of them.
+4. For APM agents, restart your service.
 
-### Tips on keeping tags simple: [#simple-tips]
+### Remove tags [#remove-tags]
 
-* Start by adding only tags you know you’ll definitely use. Unused tags create noise and may add confusion.
-* Try to use short tags. Shorter tags are easier to parse, and also the UI may sometimes truncate longer tags. (See [character limits](#formatting).)
-* When possible, use keys and values that are human-readable (for example, `region: EMEA` is better than `Param8323 : 1229072`).
-* Avoid including several values like `regions: EMEA | US | LATAM` in a single tag. We recommend using three different tags for that, like `region: emea`, `region: us`, and `region: latam`. If you want to match multiple tags, you can do that using the advanced options in the filter UI.
+1. Go to [one.newrelic.com](https://one.newrelic.com) and select a monitored entity, like a monitored app or host.
+2. Near the entity's name at the top of the page, select the **See metadata and manage tags** menu.
+3. In the menu that pops up, click the **Remove tags** toggle.
+4. Click the <Icon name="fe-x" /> next to the tag you want to remove.
+5. For APM agents, restart your service.
```

Comment:
Just like above, you need to restart your service to stop seeing a tag only for NRQL purposes.

Reply:
Same as above 🙌🏻.